### PR TITLE
Linkedinlogin

### LINKED ISSUE #2: Landing Page: LinkedIn Login

### PR DESCRIPTION
**Pull request by MichaelCalabro**
#2

### CODE REVIEW

**Review by arsalansadiq — APPROVED** [review]
LGTM

File: landing-page/src/App.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import './App.css';
```
Comment (ahmedsakr) [inline landing-page/src/App.js]: material-ui has support for integrating css in the Js file: https://material-ui.com/styles/basics/

File: landing-page/src/reportWebVitals.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = onPerfEntry => {
```
Comment (ahmedsakr) [inline landing-page/src/reportWebVitals.js:1]: do you know what these web vitals are? I know they are generated by react, but are they useful?

File: landing-page/src/App.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      
+      case flowStates.SIGN_IN:
+        flowContent = 
+          <div>
+              <Paper className="Paper" elevation={3}> (Description) Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed molestie eleifend quam nec fermentum. 
+              Donec erat nunc, mollis eu ex sit amet, tempor scelerisque nunc. 
+              Praesent lobortis sollicitudin sollicitudin. Sed pharetra sed ligula sit amet gravida. 
+              Vestibulum pretium nulla orci, at varius lorem dictum quis.</Paper>
+
+              <LinkedInLogin></LinkedInLogin>
```
Comment (ahmedsakr) [inline landing-page/src/App.js]: this could just be `<LinkedInLogin />`

File: landing-page/src/reportWebVitals.js (hunk 2 above)
Comment (MichaelCalabro) [reply landing-page/src/reportWebVitals.js:1]: yeah they are generated by react, I don't know if they are useful or not

Comment (ahmedsakr) [reply landing-page/src/reportWebVitals.js:1]: might as well toss em if they ain't useful. Less clutter.

Comment (joshcampitelli) [reply landing-page/src/reportWebVitals.js:1]: Can probably just toss this whole file `reportWebVitals.js`, unless there's a reason for it?

Comment (MichaelCalabro) [reply landing-page/src/reportWebVitals.js:1]: I tried, but then I was getting a bunch of errors. I trimmed it to what looks like the minimum needed to keep react happy

**Review by ahmedsakr — APPROVED** [review]

**Pull request merged**